src/components/Dashboard/Dashboard.tsx

```diff
@@ -80,39 +229,41 @@ export class Dashboard extends React.PureComponent<IDashboardProps, any> {
                         filter={filter}

```

Comment:
Shouldn't ActiveDashboard.CompactHorizontal be removed now that the Horizontal Compact dashboard has been removed?